README.md

```diff
@@ -24,7 +24,7 @@ Python module for reading and writing UNTL metadata records.
 Requirements
 -------------
 
-* Python ~= 2.7
+* Python ~= 3.6 - 3.7
```

Comment:
I don't think the `~=` works well with a range. I'd say maybe we should change this line to `* Python 3.6 - 3.7`